docs/apache-airflow/core-concepts/operators.rst

```diff
@@ -109,7 +109,8 @@ You can also use Jinja templating with nested fields, as long as these nested fi
     )
 
 
-.. note:: The ``template_fields`` property can equally be a class variable or an instance variable.
+.. note:: The ``template_fields`` property can equally be a class variable or an instance variable and
```

Comment:
I wonder if we want to keep the instance variable part. I don’t think we have any examples for the case, and it’s not a common case at all.

Reply:
Removed mentioning `template_fields` can be a instance variable.